internal/cmd/cmds.go

```diff
@@ -142,6 +142,21 @@ func errInvalidValue(command, param string) error {
 	return fmt.Errorf("invalid value for a parameter in '%s' command for %s parameter", strings.ToUpper(command), strings.ToUpper(param))
 }
 
+//nolint:unparam
+func errIntegerOutOfRange(command string) error {
+	return fmt.Errorf("integer out of range for '%s' command", strings.ToUpper(command))
+}
+
+//nolint:unparam
+func errWrongTypeOperation(command string) error {
+	return fmt.Errorf("wrong type operation for '%s' command", strings.ToUpper(command))
+}
+
+//nolint:unparam
+func errOverflow(command string) error {
+	return fmt.Errorf("overflow for '%s' command", strings.ToUpper(command))
```

Comment:
again, this should be a generic `overflow` error. Also, what is the difference between `errOverflow` and `errIntegerOutOfRange`? Seems the same to me. we can club the two in one.